vetiver/templates.py

```diff
@@ -0,0 +1,42 @@
+from importlib_resources import files as _files
```

Comment:
You could previously `vetiver.model_card.model_card` OR `vetiver.model_card`. The first option is no longer available, it is now `vetiver.templates.model_card` or you can still `vetiver.model_card`. I doubt anyone is using `vetiver.model_card.model_card`, so I feel pretty okay making this swap.